src/core/openthread-core-default-config.h

```diff
@@ -180,6 +180,16 @@
 #endif  // OPENTHREAD_CONFIG_MAX_CHILDREN
 
 /**
+ * @def OPENTHREAD_CONFIG_END_DEVICE_TIMEOUT
+ *
+ * The default end-device timeout value (in seconds).
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_END_DEVICE_TIMEOUT
+#define OPENTHREAD_CONFIG_END_DEVICE_TIMEOUT                    240
```

Comment:
How about calling this `OPENTHREAD_CONFIG_DEFAULT_CHILD_TIMEOUT`.
  [`otThreadSetChildTimeout()`](https://github.com/openthread/openthread/blob/master/include/openthread/thread.h#L165) provides a way to change this value at runtime.